README.md

```diff
@@ -315,6 +316,12 @@ openfaas:
   kubeconfig: "~/.kube/config" # Kubeconfig file to use (only if falcosidekick is running outside the cluster)
   # minimumpriority: "debug" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug or "" (default)
 
+wavefront:
+  endpointtype: "direct" # Wavefront endpoint type, must be 'direct' or 'proxy'. If not empty, with endpointhost, Wavefront output is enabled
+  endpointhost: "bla1.wavefront.com" # Wavefront endpoint address (only the host). If not empty, with endpointhost, Wavefront output is enabled
+  endpointtoken: "abcd-12345-cabeca-987654321" # Wavefront token. Must be used only when endpointtype is 'direct'
```

Comment:
```suggestion
  endpointhost: "" # Wavefront endpoint address (only the host). If not empty, with endpointhost, Wavefront output is enabled
  endpointtoken: "" # Wavefront token. Must be used only when endpointtype is 'direct'
```
As `endpointhost` enables the output, we should leave it empty, same for token. People often copy/paste this example (it's its purpose) I want to avoid unrelevant error messages if they don't use this output.

Reply:
fixed :D